docs/contacts/activity/logging-calls.md

```diff
@@ -0,0 +1,64 @@
+# Hubspot: How to log a call
```

Comment:
```suggestion
# How to log a call
```

Reply:
I would suggest either adding "Hubspot:" to all of the trainings or removing it from all just for consistency